analyze_headersMore_directives.gen.go

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) F5, Inc.
+ *
+ * This source code is licensed under the Apache License, Version 2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+// Code generated by generator; DO NOT EDIT.
+// All the definitions are extracted from the source code
+// Each bit mask describes these behaviors:
+//   - how many arguments the directive can take
+//   - whether or not it is a block directive
+//   - whether this is a flag (takes one argument that's either "on" or "off")
+//   - which contexts it's allowed to be in
+
+package crossplane
+
+var headersMoreDirectives = map[string][]uint{
+    "more_clear_headers": {
+        ngxHTTPMainConf | ngxHTTPSrvConf | ngxHTTPLocConf | ngxHTTPLifConf | ngxConf1More,
+    },
+    "more_clear_input_headers": {
+        ngxHTTPMainConf | ngxHTTPSrvConf | ngxHTTPLocConf | ngxHTTPLifConf | ngxConf1More,
+    },
+    "more_set_headers": {
+        ngxHTTPMainConf | ngxHTTPSrvConf | ngxHTTPLocConf | ngxHTTPLifConf | ngxConf1More,
+    },
+    "more_set_input_headers": {
+        ngxHTTPMainConf | ngxHTTPSrvConf | ngxHTTPLocConf | ngxHTTPLifConf | ngxConf1More,
+    },
+}
+
+
+func MatchHeadersMore(directive string) ([]uint, bool) {
```

Comment:
Nit: I think we should provide doc comments for all generated match functions included in the package. Perhaps it should mention which nginx module this is for (Even if it's obvious), maybe even including a module version?

Reply:
Sure, applied